root/etc/e-smith/events/actions/nethserver-cockpit_update_admins_group

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/perl
+#
+##
+## Copyright (C) 2019 Nethesis S.r.l.
+## http://www.nethesis.it - nethserver@nethesis.it
+##
+## This script is part of NethServer.
+##
+## NethServer is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License,
+## or any later version.
+##
+## NethServer is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with NethServer.  If not, see COPYING.
+##
+#
+require "/usr/libexec/nethserver/api/lib/helper_functions.pl";
```

Comment:
This is an obsolete way of including some cockpit library functions. It's now possible to write

    use NethServer::ApiTools;

Check out `/usr/libexec/nethserver/api/lib/helper_functions.pl` contents!

Reply:
solved